ProteGO/UI/Dashboard/ViewController/DashboardViewController.swift

```diff
@@ -34,6 +34,11 @@ final class DashboardViewController: UIViewController, CustomView {
         self.bindUIEvents()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        self.viewModel.updateCurrentDangerStatus()
```

Comment:
Co sądzisz o wołaniu pobierania statusu dodatkowo na `appDidBecomeActive`, oprócz w `viewWillAppear`?

Reply:
mamy juz w `willEnterForeground`

Reply:
Fakt. A może lepiej byłoby to przenieść tutaj z `AppDelegate`?